transaction/transfer.go

```diff
@@ -63,12 +70,20 @@ func (TransferProcess) HandleEvent(
 		xfer := s.Root().(*transfer)
 		xfer.ToAccountID = x.ToAccountID
 
-		s.ExecuteCommand(commands.DebitAccountForTransfer{
+		s.ExecuteCommand(commands.CheckTransferAllowedByDebitPolicy{
+			Timestamp:     time.Now(),
```

Comment:
I suspect `time.Now()` is going to become a code-smell inside the domain.  What this says to me is that we're asking if it's ok to perform this debit based on the daily-limit on the day that the transfer process receives the `StartTransfer` message, which doesn't sound like **business** logic.

Reply:
So it should be the timestamp of the transaction creation?

Reply:
I think that's the only thing that makes sense for a "daily limit", from a business perspective.

Reply:
This will be resolved by adding a timestamp to transactions.